spec/systems/tasks_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe 'Task', type: :system do
+  context 'ログインしていない時' do
+    it '/login にリダイレクトされる' do
+      visit tasks_path
+      expect(current_path).to eq login_path
+    end
+  end
+
+  context 'ログインしている時' do
+    let(:user) { create(:user) }
+
+    before do
+      login(user, by_capybara: true)
+      find('.create-button').click
+    end
+
+    context 'Title が空白の時' do
+      it '「Title can\'t be blank」と表示される' do
+        click_button 'Save'
+        expect(page).to have_content 'Title can\'t be blank'
+      end
+    end
+
+    context 'Start_at が空白の時' do
+      it '「Start_at can\'t be blank」と表示される' do
+        click_button 'Save'
+        expect(page).to have_content 'Start at can\'t be blank'
+      end
+    end
+
+    context 'Finish_at が空白の時' do
+      it '「Finish_at can\'t be blank」と表示される' do
```

Comment:
nits:
変更しなくても大丈夫です 👍 
ダブルコーテーションで良いのかなと思いました 🤝 

```suggestion
      it "「Finish_at can't be blank」と表示される" do
```